docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -591,11 +478,9 @@ Replacing nested column value:
 UPDATE users SET organization = ((organization).id, (organization).name, ((organization).office.id, 'new-office-name'))
 ----
 
-NOTE: Updating UDT-based columns requires providing a value for every column in the UDT and its child UDTs, however
-`null` can also be specified in place of nested UDT column to initialize it to `null`. Not providing full list of columns
-will cause a query validation error.
+NOTE: Updating UDT columns requires providing a value for every column in the UDT and its child UDTs unless it needs to be set to `null`. Not providing full list of columns will cause a query validation error.
 
-Inserting with Query Parameter (java only):
+Inserting with query parameter (Java only):
```

Comment:
is it really java only? the example maybe (uses `Serializable`), but this could work in principle for portable and compact from other languages as well.

Reply:
I removed the "Java only" notice and added a note for Avro support on non-Java clients. (fef8c8c2)